lib/discourse_theme/cli.rb

```diff
@@ -167,6 +184,118 @@ def run(args)
         else
           UI.info "Manage: #{client.root}/admin/customize/themes/#{theme_id}"
         end
+      elsif command == "rspec"
+        spec_path = "/spec"
+        index = dir.index(spec_path)
+
+        if index
+          spec_path = dir[index..-1]
+          dir = dir[0..index - 1]
+        end
+
+        spec_directory = File.join(dir, "/spec")
+
+        unless Dir.exist?(spec_directory)
+          raise DiscourseTheme::ThemeError.new "'#{spec_directory} does not exist"
+        end
+
+        unless Dir.exist?(DISCOURSE_THEME_TEST_TMP_DIR)
+          FileUtils.mkdir_p DISCOURSE_THEME_TEST_TMP_DIR
+        end
+
+        # Checks if the container is running
+        container_name = DISCOURSE_TEST_DOCKER_CONTAINER_NAME
+        is_running = false
+        container_exists = false
+
+        if !(
+             output =
+               execute(
+                 command: "docker ps -a --filter name=#{container_name} --format '{{json .}}'",
+               )
+           ).empty?
+          container_exists = true
+          is_running = JSON.parse(output)["State"] == "running"
+        end
+
+        basename = Pathname.new(dir).basename.to_s
+        verbose = !!args.delete("--verbose")
+        headless = !!args.delete("--headless")
+
+        if !is_running || args.delete("--rebuild")
+          if container_exists
+            execute(command: "docker stop #{container_name}")
+            execute(command: "docker rm -f #{container_name}")
+          end
+
+          execute(
+            command: <<~CMD.squeeze(" "),
+              docker run -d \
```

Comment:
This will spin up a background container, which can be quite expensive in terms of resources. I think we need to give the user an easy way to shut it down when they're done. 

In our standard docker dev environment, this is handled by `d/shutdown-dev`.

Reply:
Noted here I will add a `discourse_theme rspec --shutdown` option in a follow up PR.